rust/lib.rs

```diff
@@ -144,10 +146,6 @@ fn get_topic_schema(topic: &str, version: Option<u16>) -> Result<TopicSchema, Sc
             .ok_or(SchemaError::InvalidVersion)?
     };
 
-    if schema_metadata.schema_type != SchemaType::Json {
-        return Err(SchemaError::InvalidType);
-    }
```

Comment:
Was that intentional ?

Reply:
Originally yeah, but it was written a long time ago before we ever supported msgpack here and we just forgot to update it.